cyber-resilience-sig/minutes/2025-02-17-mom-cyber-resillience-sig.md

```diff
@@ -12,7 +12,109 @@
 | Deliverable plan | Tobie | 5 |
 | AOB | | 5 |
 
-### Participants
-
+## Participants
+* Juan Rico (Eclipse Foundation)  
+* Tobie Langel (Eclipse Foundation)
+* Timo Perala (Nokia)
+* Hendrik Ebbers (OpenElements)
+* Dan Appelquist (Samsung / OpenSSF)  
+* Tobias Heldt   
+* Tobias Frech (iJUG)  
+* Jakub Zelenka (The PHP Foundation)  
+* Adrian OSullivan (Huawei)  
+* Salve J. Nilsen (CPANSec)  
+* Lars Francke (Stackable)  
+* Georg Kunz (Ericsson)  
+* Carlo Piana (Array)  
+* Alberto Pianon (Array)  
+* Daniel Thompson-Yvetot (Tauri)  
+* Dirk-Willem van Gulik (ASF)  
+* Lothar Felten (Pyramid Computer)  
+* Alistair Woodman (Erlang Ecosystem Foundation)  
+* Jeremy Stanley (Spec Committee, OpenInfra, SPI)  
+* Marta Rybczynska (Eclipse Foundation) 
+* Andrew Katz (Bristows|Orcro)   
+* Rebecca Rumbul (Rust Foundation)  
+* Simon Phipps (SWH)  
+* Sal Kimmich (GadflyAI)  
+* Jordan Maris (OSI)  
+* Victor Roland (Obeo)  
+* Olle E. Johansson, Edvina/OWASP  
+* Daniel Dilger (Mercedes-Benz Tech Innovation)  
+* Alistair Woodman (Earlang)  
+* Dick Brooks (BCG)  
+* Gerardo Lisboa  
+* Maxim Baele (OWASP)  
+* Pierre Pochery (FreeBSD Foundation)  
+* Thierry Carrez (OpenStack)
 
 ## Minutes
+### FOSDEM & Workshop output 
+
+* All resources can be found on Github, in the [dedicated FOSDEM folder](https://github.com/orcwg/orcwg/tree/main/events/2025/FOSDEM).  
+* We plan to use [GitHub](https://github.com/orcwg/orcwg/tree/main/events) as the resource hub for the events.  
+* Feel free to provide comments on existing entries or to suggest the creation of new ones.
+
+### EC CRA Expert Group 
+
+* 60 seats \- each country has a seat \- and the rest are industry & community \- many companies and communities e.g. OpenSSF, Eclipse Foundation, Apache Software Foundation, Sovereign Tech  
+* Expert group participant members can be found [here](https://ec.europa.eu/transparency/expert-groups-register/screen/expert-groups/consult?lang=en&groupID=3967).  
+  * OSS is well represented by 5-6 orgs  
+* Dirk-Willem: EU want … high level summary posted .. of their meeting ..  Also: countries… were not that opinionated before and now large block seems to be “on our side”.  
+* Lars: half the meeting was around logistics…   
```

Comment:
I'm generally not in favor of minuting individual comments and attributing them and would rather just see summary of points discussed, but others may have different opinions. 

/cc @orcwg/steering-committee